src/sql/command.py

```diff
@@ -89,3 +87,55 @@ def connection(self):
     def result_var(self):
         """Returns the result_var"""
         return self.parsed["result_var"]
+
+    def _params_helper(self, params):
```

Comment:
what is this function for?

Reply:
Was for handling `--param` and `--use-globals`, now remove in https://github.com/ploomber/jupysql/pull/137/commits/143b1af0d9a91c88fd6596628e9eb7f97198e80b